### ISSUE TITLE
[Feature_Request] Use API Instead of dedicated destination pages

### ISSUE THREAD

**Issue by Surajit0573** [apu52/Travel_Website#281]
**Is your feature request related to a problem? Please describe.**
It's not practical to create a new page every time we need to add a destination and gather details before checkout.

**Describe the solution you'd like**
Instead, we can use an API to obtain this information. This will not only increase productivity and efficiency but also expand the range of destinations available.

**Demo Video/Screenshot (Mandatory )**
![image](https://github.com/apu52/Travel_Website/assets/111490250/e84c6b3f-6404-4774-ba27-76fbd00fe498)
![image](https://github.com/apu52/Travel_Website/assets/111490250/e94c5aaf-8599-41e5-aa08-132ab61fc194)
![image](https://github.com/apu52/Travel_Website/assets/111490250/fe66927a-3e32-49fe-9337-04bc1b4f1c5d)

***Are you contributing under any open-source program ?***
Yes, I am contributing under JWOC 2024.

**Comment by rajvinder-kaur** [comment]
can u please elaborate the feature more clearly then ill be able to work on it . assign it to me

**Comment by apu52** [comment]
> can u please elaborate the feature more clearly then ill be able to work on it . assign it to me

Do you want to work on this issue?? If yes I can assign you this under GSSOC'24

**Comment by rajvinder-kaur** [comment]
yes i want to work on it and also provide some details so i can easily work on it

**Comment by apu52** [comment]
> yes i want to work on it and also provide some details so i can easily work on it

use API to load the info , pics for the destination cards mentioned there

**Comment by apu52** [comment]
@rajvinder-kaur I have assigned you

**Comment by itsekta** [comment]
@rajvinder-kaur are you still working on it?
